package.json

```diff
@@ -11,6 +11,7 @@
   "license": "MIT",
```

Comment:
hey @mthamayor, I noticed you removed these script commands you added before. Is there a reason why?

Reply:
Hello, thank you for bring my attention to this. It's a mistake on my end. I'll work on this as soon as possible.